rmgpy/molecule/molecule.py

```diff
@@ -45,6 +45,12 @@
 import itertools
 from copy import deepcopy
 
+import element as elements
+try:
+    import openbabel
+except:
+    pass
```

Comment:
Did you add these, or are they a remnant of a rebase?

Reply:
removed